modules/ROOT/pages/update-acm.adoc

```diff
@@ -21,6 +21,11 @@ The staging environment will have a partial or full copy of the data in the prod
 
 To update your API community:
 
+. Ensure your Salesforce organizations have the "External Data Change Tracking for OData 4.0 Adapter" feature enabled.
```

Comment:
ensure > ensure that
```suggestion
. Ensure that your Salesforce organizations have the External Data Change Tracking for OData 4.0 Adapter feature enabled:
```
I think since it is all init. cap. it likely does not need quotation marks.

Reply:
Fixed.